float8_experimental/float8_tensor.py

```diff
@@ -122,7 +131,7 @@ def __tensor_flatten__(self):
         return ["_data", "_scale"], ctx
 
     @staticmethod
-    def __tensor_unflatten__(inner_tensors: Dict, metadata):
+    def __tensor_unflatten__(inner_tensors: Dict, metadata, outer_size, outer_stride):
```

Comment:
not related to this PR, but fixing it to adhere to the changes in https://github.com/pytorch/pytorch/pull/114311/files